frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -7,22 +7,30 @@ import { singularize } from "metabase/lib/formatting";
 
 import * as Lib from "metabase-lib";
 
+import {
+  BinningStrategyPickerPopover,
+  TemporalBucketPickerPopover,
+} from "./BucketPickerPopover";
+
 const DEFAULT_MAX_HEIGHT = 610;
 
-interface QueryColumnPickerProps {
+type ColumnListItem = Lib.ColumnDisplayInfo & {
```

Comment:
nit: I think in many places with call `displayInfo` inline, maybe we should do here the same

Reply:
^ updated comment

Reply:
@ranquild TBH, I'm still not sure if doing `columns.map(col => Lib.displayInfo(...))` instead of calling `displayInfo` inline is worth it. WDYT about making it bake for a little bit? We can get back to it later, all the code doing that is written in TypeScript and covered with tests, so it should be a quick and safe refactoring if we change our mind

Reply:
Yeah I changed the comment, now it's only about not spreading column into the displayInfo